### PR TITLE
feat: Integrate Footer into global layout for consistent design

### LINKED ISSUE #53: Integrate Footer into Global Layout
## 📘 Description

Currently, the footer is visible only on the landing page, which creates inconsistencies in the application’s design. To ensure a consistent user experience, the `Footer` component should also appear on the Marketplace page by integrating it into the global layout.

## 🔢 **Steps**

1. Add the `Footer` component into the global layout file of the application.
2. Remove the `Footer` import from the landing page.
3. Validate that the `Footer` displays correctly on the Marketplace page without affecting the existing layout or functionality.

### ✅ Acceptance Criteria

- The `Footer` should appear on all pages.
- Page functionality and design should remain unaffected after the implementation.

### 🌎 References

Not applicable.

### 📜 Additional Notes

⚠ **Do not apply until the ODHack begins.**
⚠ **Read our guidelines before applying.**

### PR DESCRIPTION
**Pull request by davedumto**
# 📝 Pull Request Title
feat: Integrate Footer into global layout for consistent design

## 🛠️ Issue
- Closes #53

## 📖 Description
- Integrated the Footer component into the global layout, ensuring that it appears consistently across all pages, including the Marketplace.
- Removed redundant Footer import from the landing page to streamline code and prevent duplication.
- Validated that the Footer is displaying as expected on all pages, ensuring consistency in the user interface and functionality.
- Resolved visibility inconsistencies with the Footer, improving the overall user experience.

## ✅ Changes made
- Added Footer component to global layout.
- Removed Footer import from landing page.
- Ensured the Footer displays correctly across all pages.
- Fixed Footer visibility inconsistencies.

## 🖼️ Media (screenshots/videos)
- None

## 📜 Additional Notes
- I added some padding to the component on mobile view to maintain consistency

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -16,9 +16,9 @@
 		"clsx": "^2.1.1",
 		"embla-carousel-react": "^8.5.1",
 		"lucide-react": "^0.454.0",
-		"next": "14.2.16",
```
Comment (derianrddev) [inline frontend/package.json]: Please revert the changes to the package versions in package.json (keep: `next: 14.2.16`, `react: ^18`, `react-dom: ^18`). Do not modify this file. After reverting, delete the `node_modules` folder and `run npm install` to update `package-lock.json`.

File: frontend/app/layout.tsx [hunk 2]
```diff
@@ -29,7 +30,11 @@ export default function RootLayout({
 			<body
 				className={`${geistSans.variable} ${geistMono.variable} antialiased`}
 			>
-				<Theme>{children}</Theme>
+				<Theme>
+					{children} {/* Main content */}
```
Comment (derianrddev) [inline frontend/app/layout.tsx]: Remove this comment.

File: frontend/app/layout.tsx [hunk 3]
```diff
@@ -29,7 +30,11 @@ export default function RootLayout({
...
+					{/* Added the Footer in the Layout */}
```
Comment (derianrddev) [inline frontend/app/layout.tsx]: Remove this comment.

File: frontend/app/layout.tsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import { Theme } from "@radix-ui/themes";
 import type { Metadata } from "next";
 import localFont from "next/font/local";
+import Footer from "./components/ui/footer"; //added the footer import//
```
Comment (derianrddev) [inline frontend/app/layout.tsx]: Remove this comment.

**Review by derianrddev — CHANGES_REQUESTED** [review]

File: frontend/app/layout.tsx (hunk 3 above)
Comment (davedumto) [reply frontend/app/layout.tsx]: Okay.

File: frontend/app/layout.tsx (hunk 2 above)
Comment (davedumto) [reply frontend/app/layout.tsx]: Okay

File: frontend/app/layout.tsx (hunk 4 above)
Comment (davedumto) [reply frontend/app/layout.tsx]: Okay

File: frontend/package.json (hunk 1 above)
Comment (davedumto) [reply frontend/package.json]: Done

**Comment by davedumto** [comment]
If there is any other changes kindly specify.

**Review by derianrddev — APPROVED** [review]

**Comment by danielcdz** [comment]
@davedumto please resolve the conflicts! after that, we are good to merge

**Comment by davedumto** [comment]
> @davedumto please resolve the conflicts! after that, we are good to merge

While looking at the logic, I realized the dark mode logic was removed from the Page.tsx. should I resolve it like that?

**Comment by derianrddev** [comment]
> > @davedumto please resolve the conflicts! after that, we are good to merge
> 
> While looking at the logic, I realized the dark mode logic was removed from the Page.tsx. should I resolve it like that?

The dark mode logic is now in header.tsx. The header was moved to a separate file.

**Comment by derianrddev** [comment]
image
@davedumto I've seen the conflicts; you can delete those lines.

**Comment by davedumto** [comment]
> image @davedumto I've seen the conflicts; you can delete those lines.

Okay I am doing that now

**Comment by davedumto** [comment]
I have resolved the conflicts.

**Review by danielcdz — APPROVED** [review]
LGTM
